src/hotspot/share/classfile/classFileParser.cpp

```diff
@@ -5884,6 +5945,7 @@ ClassFileParser::ClassFileParser(ClassFileStream* stream,
   _nest_members(NULL),
   _nest_host(0),
   _permitted_subclasses(NULL),
+  _preload_classes(NULL),
   _record_components(NULL),
   _temp_local_interfaces(NULL),
   _local_interfaces(NULL),
```

Comment:
Do you need to free _preload_classes in ClassFileParser::~ClassFileParser() and set _preload_classes to NULL in ClassFileParser::clear_class_metadata(), also, assert that it is NULL in ClassFileParser::fill_instance_klass() ?

Reply:
Fixed